apps/ethvm/src/components/blocks/largeBlocks/blockDetail.vue

```diff
@@ -22,7 +22,7 @@
                   <v-layout row justify-start align-center fill-height>
                     <v-card-title class="info--text pr-0 pl-0">{{ $t('common.hash') }}:</v-card-title>
                     <v-card-text class="text-truncate">
-                      <router-link :to="'/block/' + uncles[index]"> 0x{{ uncles[index].unclesHash }} </router-link>
+                      <router-link :to="'/uncle/0x' + uncles[index]"> 0x{{ uncles[index] }} </router-link>
```

Comment:
@niktrix Remove hardcoded 0x and embed that inside the model

Reply:
fixed